tests/test_client.py

```diff
@@ -302,12 +303,36 @@ def test_get_with_defaults(self):
             values, index=pd.to_datetime(index, utc=True), name="values"
         )
 
-        response = self.client.get(self.timeseries_id)
+        with patch.object(self.auth, "get") as mock_get:
+            mock_response = Mock()
+            mock_response.raise_for_status.return_value = None

```

Comment:
L. 308 doesn't seem to do anything?

Reply:
explain?

Reply:
But probably more correct to define it, since it will be called

Reply:
I tried to give different values (200, 400, 500), test still passed, is that expected behaviour? is it because mock_requests_get.raise_for_status doesn't do anything?

Reply:
Yes, that is expected since the return value is not used for anything. The `raise_for_status()` method will just be called to see if it raises any exception. But if you give the mock a side_effect where it raises one of these exceptions, the tests will probably fail.

Reply:
Good discussions. My gut feeling is that its not worth spending time on fixing fundamentally broken unit tests. We need to rewrite the whole shabang anyways.